Topics/Development_Process/Conda/Tasks_Conda.md

```diff
@@ -0,0 +1,213 @@
+## Managing Conda
```

Comment:
Maybe you can dd a general description in the beginning of the file

Reply:
Thank you for your suggestion! I splited the file into four separate files and each file introduced one kind of task. And I add a high-level description for each file at the beginning.